binary-codec/types/interfaces/binary_serializer.go

```diff
@@ -0,0 +1,8 @@
+package interfaces
+
+import "github.com/Peersyst/xrpl-go/binary-codec/definitions"
+
+type BinarySerializer interface {
```

Comment:
It could be good to have some comments to explain what the functions do.

Reply:
Absolutely. I'm going to do it in a separate PR to finish the binary-codec package